{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/sentry.py

```diff
@@ -0,0 +1,10 @@
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+from .base import env
+
+sentry_sdk.init(
+    dsn=env('SENTRY_DSN', default=''),
+    integrations=[DjangoIntegration()],
+    send_default_pii=True # If True associate users to errors
```

Comment:
Can we have this as an env variable too? This is something that may vary between deployments of the same app.